### DISCUSSION TITLE
Detail Page request

### DISCUSSION THREAD

**Discussion by podworx** [rrirower/DP2SceneGraphWizard#36] [Q&A]
Firstly, let me thank you for this software.  I've been working with a Roku "expert" for months and we've still not gotten as far as I've done with your code and a day of work.

I'd like to make a suggestion; I believe the layout of the Details Page could be a bit more useful.  Instead of a giant play button, I would swap the wides of the two LayoutGroups so the Thumbnail and description take up more room, width-wise, than the play button.  Additionally, I'd reduce the left and right margin that surrounds the first LayoutGroup.

![Capture](https://github.com/rrirower/DP2SceneGraphWizard/assets/50211012/ada528ce-9b43-498b-9a48-233fd5833113)

**Comment by rrirower** [comment]
That doesn't look like current code rendering. I don't recall seeing that in my most recent testing.

**Reply by podworx** [reply]
With the exception of more lines in the description (you had "3", I changed it to "5"), this is what your code renders on my Detail Page.

**Comment by podworx** [comment]
![image](https://github.com/rrirower/DP2SceneGraphWizard/assets/50211012/3f2b6f6d-8197-49d0-a882-efc457cce00e)

**Reply by rrirower** [reply]
What version of the code are you running?  Look in main.brs

**Reply by podworx** [reply]
' BrightScript code version 1.0.0.0-alpha-6
' https://github.com/rrirower
' Constant values.

**Reply by ferdiworks** [reply]
@rrirower Yes I see the same with version 1.0.0.0-alpha-6

**Reply by rrirower** [reply]
I'll need to research further. It's not a high priority.

**Comment by rrirower** [comment]
Please see Beta1 release.
